test/buildah-bud/apply-podman-deltas

```diff
@@ -217,6 +217,12 @@ skip_if_remote "--output option not implemented in podman-remote" \
                "build with custom build output and output rootfs to tar by pipe" \
                "build with custom build output must fail for bad input"
 
+# https://github.com/containers/podman/issues/14544
+skip_if_remote "bud-logfile-with-split-logfile-by-platform"
+
+# https://github.com/containers/podman/issues/14547
+skip_if_remote "build-with-omit-history-to-true should not add history"
```

Comment:
@edsantiago I don't understand why the two tests are not skipped in the remote bud tests. Can you take a look?

Reply:
Improperly written. Please look at the treadmill PR to see how it's done. Sorry for terseness, I'm incredibly swamped right now.

Reply:
Whew. Crisis resolved. First argument to `skip` is an explanation; subsequent args are test names:
```bash
# FIXME FIXME FIXME: this should be easy for someone to wire up
skip_if_remote "FIXME FIXME FIXME: omit-history not implemented in remote" \
               "build-with-omit-history-to-true should not add history"

# FIXME FIXME FIXME this too
skip "FIXME FIXME FIXME this should be easy for someone to fix" \
     "bud-logfile-with-split-logfile-by-platform"
```
The usage you wrote should trigger a loud failure. That is my fault, lack of foresight. I will fix that in a followup PR.

Reply:
Aaaaah, my bad. Thanks a lot, Ed!